elyra/metadata/schemas/airflow.json

```diff
@@ -127,6 +127,7 @@
         "cos_username",
         "cos_password",
         "cos_bucket",
+        "github_api_endpoint",
```

Comment:
This introduces a duplication, see two items below. This is what is causing the test failure

Reply:
fixed